api/src/pcapi/routes/public/individual_offers/v1/events.py

```diff
@@ -611,3 +611,31 @@ def patch_event_date(
     except offers_exceptions.OfferCreationBaseException as error:
         raise api_errors.ApiErrors(error.errors, status_code=400)
     return serialization.DateResponse.build_date(edited_date)
+
+
+@blueprint.v1_blueprint.route("/events/categories", methods=["GET"])
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** /‏subcategories

Dans la route ici on fait en résumé:
GET /categories -> returns subcategories

Les catégories, sous-catégories et autres catégories natives sont déjà facilement confusante, je suggère de garder le nom de `subcategory` quand on retourne des sous-catégories

Reply:
Vu ensemble: le nom ici est à destination des 3rd parties qui appelent notre API.
On choisit de laisser `categiry` dans tout ce qui apparait dans la doc (i.e.) le swagger)
Et on ajoute un commentaire pour expliquer le décalage de naming

Je marque résolu